### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by mifyy**

### CODE REVIEW

**Comment by mifyy** [comment]
https://www.loom.com/share/934e85fe5e1a4391b239cf1fe57ed75b?sid=584000bd-0bdb-450b-8489-7ad589921b73

**Comment by mifyy** [comment]
I had to hard-code this part because I could not manage to implement it otherwise. Maybe you can offer a better solution

  .footer,
  .footer__dark {
    justify-content: flex-start;
    padding: 32px 0;
    margin: 0;
    flex-direction: column;
    height: 175px;
  }

File: src/modules/shared/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -1,7 +1,49 @@
...
+      <div className={styles.container}>
+        <Link to="/">
+          <img
+            src={isDarkTheme ? logo_dark : logo}
+            alt="nice_gadgets_logo"
+            className={styles.nav_logo}
+          />
+        </Link>
+        <div className={styles.nav_center}>
+          <Link to="/github" className={`${isDarkTheme ? styles.nav_text__dark : styles.nav_text}`}>
```
Comment (mpohorenyi) [inline src/modules/shared/components/Footer/Footer.tsx:23]: It should redirect to our repository on GitHub.

File: src/modules/shared/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -1,7 +1,49 @@
...
+            Github
+          </Link>
+          <Link to="/contacts" className={`${isDarkTheme ? styles.nav_text__dark : styles.nav_text}`}>
+            Contacts
+          </Link>
+          <Link to="/rights" className={`${isDarkTheme ? styles.nav_text__dark : styles.nav_text}`}>
+            Rights
+          </Link>
```
Comment (mpohorenyi) [inline src/modules/shared/components/Footer/Footer.tsx:31]: It should redirect to the home page `'/'` This is temporary.

File: src/modules/shared/components/Footer/Footer.module.scss [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    flex-direction: column;
+    gap: 16px;
+  }
+
+  .return_button,
+  .return_button__dark {
+    align-self: center;
+    justify-content: center;
+  }
+}
```
Comment (mpohorenyi) [inline src/modules/shared/components/Footer/Footer.module.scss:137]: Our project has a mixin for this purpose

File: src/modules/shared/components/Footer/Footer.module.scss [hunk 4]
```diff
@@ -0,0 +1,137 @@
+@font-face {
+  font-family: 'Mont';
+  src: url('../../../../fonts/Mont-Regular.ttf');
+}
```
Comment (mpohorenyi) [inline src/modules/shared/components/Footer/Footer.module.scss:4]: Fonts are already connected to the entire project

**Review by mpohorenyi — CHANGES_REQUESTED** [review]
Good job, please pay attention to the comments. 

You have the wrong @media to phone resolution, look at the layout carefully, there are 320-639

Please rework your styles according to BEM.

File: src/modules/shared/components/Footer/Footer.tsx (hunk 2 above)
Comment (mpohorenyi) [reply src/modules/shared/components/Footer/Footer.tsx:31]: UPD: Need these links to redirect to the "Page in Progress"

**Pull request merged**